app/views/shared/_glossary_hover.html.erb

```diff
@@ -1,4 +1,7 @@
-<span id= "<%= key %>" class="glossary"
+<!-- added random string to make duplicate instances on a page work -->
+<% key_index = local_assigns[:key_index] ? key_index : "" %>
```

Comment:
do we need another parameter for this partial, or can we just modify the key at the call site as an argument? like `render partial:"shared/glossary_hover", locals: {key: "generic_make_payments_hover_#{key_index}", title: "Make payments for my plan", term: sanitize(l10n("plans.issuer.pay_now.make_payments"))

Reply:
no, because the key is used to look up the translation string